README.md

```diff
@@ -32,6 +32,21 @@ To use the xAPI Java Client include the appropriate XML in the `dependencies` se
 </project>
 ```
 
+### Configuration
+
+The xAPI Java Client has a Spring AutoConfiguration bean which picks up the following properties from the environment:
+
+```
+xapi.client.baseUrl
+xapi.client.username
+xapi.client.password
+xapi.client.authorization
+```
+
+Set `xapi.client.baseUrl` to your xApi LRS endpoint. Use `xapi.client.username` and `xapi.client.password` for basic authorization, or set the whole authorization header via `xapi.client.authorization`. (The latter has precedence if both method is used)
```

Comment:
Link to spring documentation for how to set properties. (see other pro for example.)